src/plugins/loan-channel-query-map.js

```diff
@@ -11,7 +11,7 @@ export default {
 				{
 					id: 5,
 					url: 'women',
-					queryParams: 'status=fundRaising&gender=female',
+					queryParams: 'status=fundRaising&gender=female&distributionModel=both',
```

Comment:
Just out of curiosity, did you manually walk through the Kiva Admin settings per channel, or is there a way to automate the export of channel filters?

Reply:
> Just out of curiosity, did you manually walk through the Kiva Admin settings per channel, or is there a way to automate the export of channel filters?
@dyersituations I used a portion of flssLoanChannel query to get the loan channels in graphql